scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/RestResponseController.java

```diff
@@ -63,6 +63,24 @@ public ResponseEntity<NotificationDto> getByName(
         .orElseGet(() -> new ResponseEntity<>(HttpStatus.NOT_FOUND));
   }
 
+  @DeleteMapping("/run/{name}")
+  @ApiOperation(
+      value = "Invalidate cache for run by name",
+      response = Boolean.class,
+      responseContainer = "ResponseEntity")
+  @ApiResponses({
+    @ApiResponse(code = 200, message = "Success"),
+    @ApiResponse(code = 404, message = "Run not found")
+  })
+  public ResponseEntity<Boolean> deleteByName(
```

Comment:
This isn't actually returning a boolean value, but I don't think it should anyway since the response status already indicates the result. Should return 204 (no content) for success though

Reply:
Swagger does say "Resolver error at paths./runscanner/run/{name}.delete.responses.204.schema.$ref
Could not resolve reference: Could not resolve pointer: /definitions/Error-ModelName{namespace='java.lang', name='Void'} does not exist in document" when I do this though